### ISSUE TITLE
Support recursive type

### ISSUE THREAD

**Issue by xerial** [wvlet/airframe#32]
Binding to finagle's Service shows warning:

```
class Service[-Req, +Rep] extends (Req => Future[Rep]) 

2016-09-19 13:29:59-0700  warn [wvlet.obj.ObjectSchema] No corresponding class for scala.<repeated> is found  - (ObjectSchema.scala:566)
```

**Comment by xerial** [comment]
Another example:
```
2016-09-19 13:33:41-0700 trace [wvlet.obj.ObjectSchema] resolveClassFromName(com.twitter.util.Closable, ClassSymbol(Closable, owner=com.twitter.util, flags=2000080, info=9 ,None))  - (ObjectSchema.scala:523)
2016-09-19 13:33:41-0700 trace [wvlet.obj.ObjectSchema] resolveObjectType: [Closable raw:Closable] <= TypeRefType(ThisType(com.twitter.util),ClassSymbol(Closable, owner=com.twitter.util, flags=2000080, info=9 ,None),List())  - (ObjectSchema.scala:644)
2016-09-19 13:33:41-0700 trace [wvlet.obj.ObjectSchema] resolveObjectType: [Object[Closable] raw:Object[Closable]] <= TypeRefType(ThisType(scala),scala.<repeated>,List(TypeRefType(ThisType(com.twitter.util),ClassSymbol(Closable, owner=com.twitter.util, flags=2000080, info=9 ,None),List())))  - (ObjectSchema.scala:644)
2016-09-19 13:33:41-0700 trace [wvlet.obj.ObjectSchema] method: (MethodSymbol(sequence, owner=46, flags=200, info=85 ,None),MethodType(TypeRefType(ThisType(com.twitter.util),ClassSymbol(Closable, owner=com.twitter.util, flags=2000080, info=9 ,None),List()),List(MethodSymbol(closables, owner=83, flags=2000, info=80 ,None))))  - (ObjectSchema.scala:456)
2016-09-19 13:33:41-0700 trace [wvlet.obj.ObjectSchema]
method param refs:
(closables,TypeRefType(ThisType(scala),scala.<repeated>,List(TypeRefType(ThisType(com.twitter.util),ClassSymbol(Closable, owner=com.twitter.util, flags=2000080, info=9 ,None),List()))))  - (ObjectSchema.scala:303)
2016-09-19 13:33:41-0700 trace [wvlet.obj.ObjectSchema] Alias type: TypeRefType(ThisType(scala),scala.<repeated>,List(TypeRefType(ThisType(com.twitter.util),ClassSymbol(Closable, owner=com.twitter.util, flags=2000080, info=9 ,None),List()))), path:scala.<repeated>  - (ObjectSchema.scala:630)
2016-09-19 13:33:41-0700 trace [wvlet.obj.ObjectSchema] resolveClassFromName(scala.<repeated>, scala.<repeated>)  - (ObjectSchema.scala:523)
2016-09-19 13:33:41-0700  warn [wvlet.obj.ObjectSchema] No corresponding class for scala.<repeated> is found  - (ObjectSchema.scala:566)
java.lang.ClassNotFoundException: scala$<repeated>```

**Comment by xerial** [comment]
Implementation in #44 has no problem like that.

**Comment by xerial** [comment]
Fixed by #44
